src/bin/options/grep.ts

```diff
@@ -0,0 +1 @@
+export { applyGrepPattern as default } from '../../util';
```

Comment:
I usually try to keep all supported option handlers in this directory so that we know what's supported and what's not. Not sure if this is the right idea, however it's quite obvious for now 🤷‍♂️